scripts/enforce_header.py

```diff
@@ -1,27 +1,29 @@
-from datetime import datetime
+from __future__ import division, print_function, absolute_import
```

Comment:
I'm actually not sure *why* it's not enforced. I think it's a good idea to have it though, yeah. Will fix.